File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.py
def solution(rsp):
    # 딕셔너리로 이기는 경우를 짝지어 놓는다
    dic = {'0': '5', '2': '0', '5': '2'}
    
    # 방법 2 - 리스트 컴프리핸션, join 사용
    return ''.join([dic[el] for el in rsp])
    
    # # 방법 1 - map, lambda, join 사용
    # return ''.join(map(lambda x: dic[x], rsp))

File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    // for문, 삼항 연산자 사용.
    let answer = ''
    
    for (let el of rsp) {
        answer += el === '2' ? '0' : el === '0' ? '5' : '2'
    }
    
    return answer
}